### DISCUSSION TITLE
Drag and drop to reorder tabs

### DISCUSSION THREAD

**Discussion by Raytle** [hluk/CopyQ#1709] [Ideas]
To reorder tabs I would like a feature that allows dragging and dropping a tab title. 

For example, to reorder the "Text Clipboard" depicted below, from the bottom (that is, the second position), to the top (that is, the first position), I would like to use my mouse to drag "Text Clipboard" to a position above "Images Clipboard."

![Image 1](https://user-images.githubusercontent.com/81956552/125210714-6ab14780-e256-11eb-914a-d0b584701bea.png)

Better till, I would like to use my mouse to drag "Text Clipboard" to a position to the left of the "Images Clipboard."...

![Image 2](https://user-images.githubusercontent.com/81956552/125210874-46a23600-e257-11eb-8902-05fc0a30a0d9.png)

**Comment by hluk** [comment]
Good idea. Adding the feature:  #1716

(This request would have been better as a GitHub issue - [new feature request](https://github.com/hluk/CopyQ/issues/new?assignees=&labels=feature&template=feature_request.md&title=) - rather than a discussion.)
